ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/recovery/arjunacore/XARecoveryModule.java

```diff
@@ -78,6 +78,16 @@
 
 public class XARecoveryModule implements ExtendedRecoveryModule
 {
+	private static AtomicBoolean recoveryProblems = new AtomicBoolean(false);
+
+	public static boolean isRecoveryProblems() {
```

Comment:
if there is no particular reason to make anything public then it should be `private` (or package-private in case)

Reply:
I will remove public access modifier.